### DISCUSSION TITLE
Fix outdated/misleading infos in 3rd platforms

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#78] [Integration]
Such as coingeco, CoinMarketCap etc.

Especially the descriptions and narratives, mistakes etc. The info should keep consistency and same with the website https://darwinia.network/

Please name other 3rd platforms you know but not listed here. 

We may need to do a comprehensive health check. @1022ranji should be able to co-working on this and provide necessary support during update process.

**Comment by SasoLithops** [comment]
Here are some:

https://cryptoslate.com/products/category/price-trackers/
https://www.cointracker.io/

**Comment by nblogist** [comment]
Hi @hackfisher  given the de facto status of CoinGecko and CMC in the industry, I'll reach out to update those. 
I do think that we need once consistent narrative, even between the docs and the about section the narratives are leaning between community and helping dapps.

Can you please share a short summary draft of how you want description to be positioned, I'd refine, finalize and request to publish. :)

**Reply by hackfisher** [reply]
@1022ranji Would you please prepare the draft consist with our latest content and website? We may need a review process/meeting before request publish.

**Reply by nblogist** [reply]
[92](https://github.com/orgs/dcdao/discussions/92) @1022ranji This should help you get started!

**Reply by 1022ranji** [reply]
Information was provided in discussion #92.

**Comment by nblogist** [comment]
https://github.com/orgs/dcdao/discussions/92#discussioncomment-8505061
https://github.com/orgs/dcdao/discussions/92#discussioncomment-8512916
https://github.com/orgs/dcdao/discussions/92#discussioncomment-8513324
https://github.com/orgs/dcdao/discussions/92#discussion-6226039

Notes from #92

**Comment by nblogist** [comment]
## Darwinia Network
- [x] CoinGecko
![image](https://github.com/dcdao/discussion/assets/35498810/2a07f846-4fcd-4856-b53e-98fe6a5b14c5)
- [x] CryptoSlate: Made a report and sent!
![image](https://github.com/dcdao/discussion/assets/35498810/bc70c477-3f16-4a24-b8bd-127568da7639)
- [x] ParaChains.info:  
![image](https://github.com/dcdao/discussion/assets/35498810/1b201c95-117e-4aaa-a004-fd2398335dfe)

- ❌ CoinMarketCap [**PENDING**] :
CMC requires several steps to update, including verification from the official twitter account and usage of official Darwania emails: [Form](https://support.coinmarketcap.com/hc/en-us/requests/new?ticket_form_id=360000563011) then select:  "8 - [Self-reporting Dashboard] Cryptoassets/KOLs/Organizations (official representatives only)"  @hackfisher pls lmk how you want me to proceed!

## Crab Network
@1022ranji I feel like the information present on parachain.info about crab is okay and doesn't need updates, if you think otherwise pls lmk and I'll put in a request. [Here](https://parachains.info/details/darwinia_crab/)

**Reply by 1022ranji** [reply]
> CMC 需要几个步骤来更新，包括从官方 Twitter 帐户进行验证以及使用官方 Darwania 电子邮件：填写表格，[然后](https://support.coinmarketcap.com/hc/en-us/requests/new?ticket_form_id=360000563011)选择：“8 - [自我报告仪表板] 加密资产/KOL/组织（仅限官方代表） ” @hackfisher请告诉我你希望我如何继续！
@nblogist I'll be sorting out the CMC and CoinGecko stuff all together next week.

**Reply by nblogist** [reply]
Thank you @1022ranji

**Reply by 1022ranji** [reply]
Update(both CMC and CoinGecko):
<img width="1237" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/5a28377c-fdf1-4b38-bcde-4efc59ee43f5">
<img width="1352" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/cb12840e-4ee6-4163-929a-a8871012602e">

**Comment by nblogist** [comment]
@hackfisher update: Parachain.info sent me a portal to update, which I have done and after [this](https://github.com/orgs/dcdao/discussions/92#discussioncomment-8542664) I'll submit for update.
CMC didn't let me update as described in the comment above, and CoinGecko just rejected the request on grounds of proving official representation from the project.
![image](https://github.com/dcdao/discussion/assets/35498810/e5c483a1-1b96-4e0e-9f74-468d538514f4)

Verification guide is [here](https://support.coingecko.com/hc/en-us/articles/23725417857817-Verification-Guide-for-Listing-Update-Requests-on-CoinGecko), LMK who will be making the post and I'll coordinate myself.

**Comment by 1022ranji** [comment]
The content for Darwinia on CMC has been fully updated, and on CoinGecko, everything except the category has been updated as well. The category part requires a separate form submission for review, which has been submitted and is currently awaiting approval. cc @nblogist @hackfisher

**Reply by nblogist** [reply]
Thank you so much @1022ranji !

**Reply by 1022ranji** [reply]
Now the content for Darwinia on CoinGecko has been fully updated.
